source/security.txt

```diff
@@ -0,0 +1,41 @@
+.. _security:
+
+=======================================================================
+Securely Store an {+atlas-cli+} Configuration File
+=======================================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The {+atlas-cli+}, like most popular CLIs, enables you to store your configuration file in 
+a secure manner. The following methods are available for securely storing your |service| credentials 
+and configuration file. The configuration file is only accessible to the current user.
+
+Overriding the Private API Key Value by Using an Environment Variable
+---------------------------------------------------------------------
+
+You can specify configuration settings by using :ref:`environment variables <atlas-cli-env-vars>`. 
+When you run any command, each setting that you specify with an environment variable takes precedence 
+over the profile stored in the configuration file. 
+
+You can encrypt the value of your private API key by overriding your private API key value through 
+the :ref:`MONGODB_ATLAS_PRIVATE_API_KEY environment variable <atlas-cli-env-vars>`.
+
+Setting the Location of the Configuration File Within the Secure Store
+----------------------------------------------------------------------
+
+.. note::
+   You need to use MacOS, Linux, or Windows Subshell for Linux Systems to follow the instructions below. If 
+   you are using a different operating system, the following methods are available:
+   
+   - `Creating and modifying environment variables in PowerShell <https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.3>`__
+   - Installing the ``platformdirs`` `Python package <https://pypi.org/project/platformdirs/>`__ to determine appropriate platform-specific directories
+
```

Comment:
not really sure what to do with this, what do I do once I found the file location?